src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -71,26 +115,27 @@ void ScatterUpdate::getSupportedDescriptors() {
         OPENVINO_THROW(errorPrefix, " has incorrect number of input edges");
     if (getChildEdges().empty())
         OPENVINO_THROW(errorPrefix, " has incorrect number of output edges");
-
-    if (getInputShapeAtPort(DATA_ID).getRank() < 1 ||
-        getInputShapeAtPort(INDICES_ID).getRank() < 1 ||
-            getInputShapeAtPort(UPDATE_ID).getRank() < 1) {
-        OPENVINO_THROW(errorPrefix, " do not support scalar input");
-    }
-
+    // In ov::PartialShape with rank 0 (scalars) is converted to ov::intel_cpu::Shape with rank 1.
+    // Create workaround by extracting information about rank directly from OV node.
+    bool is_not_supported_input = ov::is_scalar(ovOp->get_input_partial_shape(0)) || ov::is_scalar(ovOp->get_input_partial_shape(1));
     Type scatterUpdateType = getType();
     if (scatterUpdateType == Type::ScatterUpdate) {
         scatterUpdateMode = ScatterUpdateMode::ScatterUpdate;
         axisRelaxed = true;
+        is_not_supported_input = is_not_supported_input || ov::is_scalar(ovOp->get_input_partial_shape(2));
     } else if (scatterUpdateType == Type::ScatterElementsUpdate) {
         scatterUpdateMode = ScatterUpdateMode::ScatterElementsUpdate;
         axisRelaxed = true;
+        is_not_supported_input = is_not_supported_input || ov::is_scalar(ovOp->get_input_partial_shape(2));
     } else if (scatterUpdateType == Type::ScatterNDUpdate) {
         scatterUpdateMode = ScatterUpdateMode::ScatterNDUpdate;
         axisRelaxed = false;
     } else {
         OPENVINO_THROW(errorPrefix, " is not supported");
     }
+    if (is_not_supported_input) {
+        OPENVINO_THROW(errorPrefix, " do not support scalar input");
```

Comment:
It's better to use THROW_CPU_NODE_ERR()

Reply:
Changed to THROW_CPU_NODE_ERROR for all occurrences in file.